### PR TITLE
Update automatic deletion of past scheduled interviews

### LINKED ISSUE #140: Add automatically remove past interview dates
Currently the schedule interview details are not remove when the date has passed.
Let's enable the system to automatically remove them to reduce clutter in the list.

### PR DESCRIPTION
**Pull request by LeeEnEn**
Brief summary of what this feature does:
- Deletes interviews on startup of the application only
- Deletes interviews that are 31 mins ago, since each interview time slot is 30 minutes long

Closes #140

### CODE REVIEW

File: src/main/java/seedu/address/MainApp.java [hunk 1]
```diff
@@ -183,6 +184,7 @@ protected UserPrefs initPrefs(UserPrefsStorage storage) {
     @Override
     public void start(Stage primaryStage) {
         logger.info("Starting AddressBook " + MainApp.VERSION);
+        model.deletePastInterviewsForInterviewList(LocalDateTime.now());
```
Comment (domlimm) [inline src/main/java/seedu/address/MainApp.java:187]: 💯 / 💯

Comment (LeeEnEn) [reply src/main/java/seedu/address/MainApp.java:187]: 😘

**Review by lzan98 — APPROVED** [review]
LGTM! 👍 Well done and thank you <3

**Review by domlimm — APPROVED** [review]
LGTM! Really smart way of implementing this! Good job! 💯

**Pull request merged**